### PR TITLE
Feature/udp

### PR DESCRIPTION
**Pull request by ewoudje**
https://github.com/ValkyrienSkies/vs-core/pull/23
Closes https://github.com/ValkyrienSkies/Valkyrien-Skies-2/issues/91

### CODE REVIEW

File: common/src/main/java/org/valkyrienskies/mod/mixin/client/world/MixinClientLevel.java [hunk 1]
```diff
@@ -16,23 +17,41 @@
...
+    @Unique
+    private boolean serverNoUdp = false;
+    @Unique
+    private int tryConnectIn = 100;
+
     @Inject(method = "tick", at = @At("HEAD"))
     private void preTick(final BooleanSupplier shouldKeepTicking, final CallbackInfo ci) {
         // Drag entities
         EntityDragger.Companion.dragEntitiesWithShips(entitiesById.values());
+        if (!VSNetworking.INSTANCE.getClientUsesUDP() && !serverNoUdp) {
```
Comment (StewStrong) [inline common/src/main/java/org/valkyrienskies/mod/mixin/client/world/MixinClientLevel.java:43]: Can we move this logic to ShipObjectWorldClient?

Comment (ewoudje) [reply common/src/main/java/org/valkyrienskies/mod/mixin/client/world/MixinClientLevel.java:43]: Hmm might be better yes

Comment (ewoudje) [reply common/src/main/java/org/valkyrienskies/mod/mixin/client/world/MixinClientLevel.java]: Fixed but became also quite.. ugly

**Comment by StewStrong** [comment]
I'm going to add encryption to this.

**Pull request merged**